stable/traefik/README.md

```diff
@@ -87,7 +87,9 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `acme.persistence.size`         | Minimum size of the volume requested                                 | `1Gi`                                     |
 | `dashboard.enabled`             | Whether to enable the Traefik dashboard                              | `false`                                   |
 | `dashboard.domain`              | Domain for the Traefik dashboard                                     | `traefik.example.com`                     |
-| `gzip.enabled`                  | Whether to use gzip compression                                      | `true`                     |
+| `gzip.enabled`                  | Whether to use gzip compression                                      | `true`                                    |
+| `kubernetes.namespaces`         | List of Kubernetes namespaces to watch (e. g. `["default"]`)         |                                           |
```

Comment:
Ok... this has to change. I didn't read this carefully before. I just saw the word "list" and naturally did this:

```
kubernetes:
  namespaces:
  - foo
  - bar
```

^ That's the native YAML way of expressing a list. Requiring a format like `["default"]` feels very awkward because it's mixing TOML syntax with YAML syntax.